docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+services:
+  db:
+    image: mdillon/postgis
+  web:
```

Comment:
You need to add a `volumes` section to this and a `working_dir` section. For instance:
```
volumes:
  - .:/path/to/code
working_dir: /path/to/code
command: python /path/to/code/manage.py runserver 0.0.0.0:800
```